test/mason/mason-test-exit/SKIPIF

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
One minor suggestion: We tend to use symlinks for `EXECENV`, and `SKIPIF` files in the mason testing to reduce the number of changes required if we ever need to modify them.